src/Modules/CalcPerform.lua

```diff
@@ -481,10 +481,17 @@ local function doActorAttribsPoolsConditions(env, actor)
 		end
 		if actor.mainSkill.skillFlags.hit and not actor.mainSkill.skillFlags.trap and not actor.mainSkill.skillFlags.mine and not actor.mainSkill.skillFlags.totem then
 			condList["HitRecently"] = true
+			if actor.mainSkill.skillFlags.spell then
+				condList["HitSpellRecently"] = true
+			end
 		end
 		if actor.mainSkill.skillFlags.totem then
 			condList["HaveTotem"] = true
+			condList["TotemsHitRecently"] = true
```

Comment:
You'll want to put this behind a hit check (see the block right above this one) otherwise things like Scorching Ray totems will count.  Same with the SpellHitRecently condition below